jsdecode/decode_json_test.go

```diff
@@ -1,13 +1,15 @@
 package jsdecode
 
+/*
```

Comment:
I'm not sure why the everything is commented out in this file with this change. Should the change to this file be reverted?